### PR TITLE
Simple fix

### PR DESCRIPTION
**Pull request by msaxena2**
Fixes to CSV to follow new convention.

### CODE REVIEW

**Comment by dwightguth** [comment]
@msaxena2 can you fix this rapidly so that it can be merged? we are blocked waiting for all changes to the examples/simple directory to be merged and I'd like us to get this done as fast as possible.

**Comment by dwightguth** [comment]
Can you please fix the conflicts on this? I'd like to get this merged so that I can do the rename on the directory that Grigore wants.

**Comment by msaxena2** [comment]
@dwightguth Please re-review.

File: examples/simple/Error_Codes.csv [hunk 1]
```diff
@@ -169,4 +169,4 @@ L-STDLIBE3,"Realloc overflows memory bound, returning null pointer.",,Possible u
 L-CEIE1,Memory malloced but never freed.,,Possible unintended behavior,X
 L-CCVE1,unsigned integer overflow.,,Possible unintended behavior,X
 L-CCVE2,conversion to unsigned integer outside the range that can be represented.,,Possible unintended behavior,X
-L-EXC3,Comparing a function pointer to a null pointer constant.,,Possible unintended behavior,X
+L-EXC3,Comparing a function pointer to a null pointer constant.,,Possible unintended behavior,X
```
Comment (dwightguth) [inline examples/simple/Error_Codes.csv]: can you fix this please? noeol is generally bad for most purposes with text files.

**Comment by dwightguth** [comment]
lgtm. I will merge this if the tests pass

**Pull request merged**